### PR TITLE
add unit test for semi_auto_llama accuary aligment.

### PR DESCRIPTION
**Pull request by winter-wang**
### PR Category

Auto Parallel

### PR Types

Others

### Description

- add unit test for semi_auto_llama accuary aligment.
### Other 
Pcard-67164

### CODE REVIEW

File: test/auto_parallel/hybrid_strategy/semi_auto_llama_acc_align.py [hunk 1]
```diff
@@ -91,11 +92,13 @@ def __init__(self):
         if os.getenv("use_sp") == "true":
             self.config.sequence_parallel = True
         self.gradient_accumulation_steps = int(os.getenv("acc_step"))
+        self.config.recompute = int(os.getenv("recompute"))
```
Comment (JZ-LIANG) [inline test/auto_parallel/hybrid_strategy/semi_auto_llama_acc_align.py]: 重计算当前版本还不支持

File: test/auto_parallel/hybrid_strategy/semi_auto_llama_acc_align.py [hunk 2]
```diff
@@ -91,11 +92,13 @@ def __init__(self):
...
+        self.config.sep_parallel_degree = 1
 
         self.init_dist_env()
 
     def init_dist_env(self):
-        order = ["dp", "pp", "mp"]
+        order = ["pp", "dp", "mp"]
```
Comment (JZ-LIANG) [inline test/auto_parallel/hybrid_strategy/semi_auto_llama_acc_align.py]: 这里应该对精度没有影响

**Review by JZ-LIANG — DISMISSED** [review]
LGTM

File: test/auto_parallel/hybrid_strategy/semi_auto_llama_acc_align.py (hunk 2 above)
Comment (winter-wang) [reply test/auto_parallel/hybrid_strategy/semi_auto_llama_acc_align.py]: done, thanks!

File: test/auto_parallel/hybrid_strategy/semi_auto_llama_acc_align.py (hunk 1 above)
Comment (winter-wang) [reply test/auto_parallel/hybrid_strategy/semi_auto_llama_acc_align.py]: done, thanks!

**Review by JZ-LIANG — APPROVED** [review]
LGTM

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Review by XieYunshen — APPROVED** [review]

**Pull request merged**